### PR TITLE
🔨 Replace legacy POST `/txs` 

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
   "dependencies": {
     "@cosmjs/launchpad": "^0.23.1",
     "@cosmjs/launchpad-ledger": "^0.23.1",
```
Comment (williamchong) [inline package.json]: please upgrade to `^0.28.3` also

File: signing/transaction-manager.js [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
+import { TxRaw } from 'cosmjs-types/cosmos/tx/v1beta1/tx'
+import { BroadcastMode } from 'cosmjs-types/cosmos/tx/v1beta1/service'
 import axios from 'axios'
 import { getSigner } from './signer'
 import messageCreators from './messages.js'
 import fees from '~/common/fees'
 import network from '~/common/network'
 import { signWithExtension } from '~/common/extension-utils'
 
+const prefix = 'cosmos'
```
Comment (williamchong) [inline signing/transaction-manager.js]: What is the meaning for this `prefix` ? Message prefix?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (WeiJun0827) [reply package.json]: Done!

File: signing/transaction-manager.js (hunk 2 above)
Comment (WeiJun0827) [reply signing/transaction-manager.js]: It's the address prefix, already referenced from network config

**Review by williamchong — APPROVED** [review]

**Pull request merged**